blazor/multiselect-dropdown/checkbox-grouping.md

```diff
@@ -48,4 +48,26 @@ You can arrange the datasource items by grouping them with checkbox mode in Mult
 
 ```
 
-![Blazor MultiSelect DropDown with Checkbox Grouping](./images/blazor-multiselect-dropdown-checkbox-grouping.png)
\ No newline at end of file
+![Blazor MultiSelect DropDown with Checkbox Grouping](./images/blazor-multiselect-dropdown-checkbox-grouping.png)
+
+## Properties
+
+### EnableGroupCheckBox 
+
+Specifies a boolean value that indicates the whether the grouped list items are allowed to check by checking the group header in checkbox mode.
```

Comment:
Specifies a Boolean value that enables checkbox in the group header in checkbox mode.